orttraining/orttraining/training_ops/cpu/optimizer/gradient_control.cc

```diff
@@ -44,17 +58,9 @@ Status InPlaceAccumulator<T>::Compute(OpKernelContext* context) const {
     }
   }
 
-  //Copy from Add CPU kernel
```

Comment:
the temporary fix on line 45 under #ifdef ENABLE_TRAINING_ON_DEVICE block can be removed now